Rettelse PR (#62)
MrLongbottom/P9-Rapport: sections/related_works.tex

\section{Related Work}\label{sec:related-works} \vejleder[inline]{add some more}
In this section, we investigate existing literature concerning \gls{lda} and similar topic models, as well as relevant work related to random walk.

> \citet{lda} describes a generative statistical model, which is heavily used within topic modeling. 

Author: ```suggestion
\citet{lda} describes \acrlong{lda}; a generative statistical model, which is widely used within the field of topic modeling. 
```

The intuition behind the \gls{lda} topic model is that a document has an underlying distribution of topics, and that the words of a document are based on these topics.

> This model is used widely within the field of topic modeling and is somewhat considered a standard, see section \autoref{sec:lda} for more detail.

Author: ```suggestion
We provide an overview of this model in \autoref{sec:lda}.
```

There also exists a large amount of extensions to \gls{lda}.

> \citet{blei2012topicmodels} describes some of the most significant extensions that are made from relaxing the assumptions of \gls{lda}.

Author: perhaps we should also mention all of these assumptions in our lda section to clarify its weaknesses.

> Some extensions mentioned are: the dynamic topic model\cite{blei2006dynamic} which assumes that topics change over time, and thus respects the order of the documents, and the correlated topic model\cite{blei2007correlated} and pachinko allocation model\cite{li2006pachinko} which each allow correlations to exist between topics.

Reviewer: ```suggestion
An extension to the \gls{lda} is the dynamic topic model\cite{blei2006dynamic} which assumes that topics change over time, and thus respects the order of the documents. 
The correlated topic model\cite{blei2007correlated} and pachinko allocation model\cite{li2006pachinko} are other variations of the \gls{lda} which each allow correlations to exist between topics.
```

\citet{quanti} describes a method of how to quantitatively preprocess and analyze large amounts of journalistic data dating from 1945-2000. 
They propose \gls{lda} as a way of categorizing articles and using this as the basis for search.
However, they only check for a single subject, namely the nuclear topic.  

\citet{Tang2008} successfully combine the two research fields of Topic Modeling and Random Walk search. 
Though their focus is mainly on mapping author and venue relations as part of their topic model, their methods for combining these two fields are still novel and generally applicable.

> \citet{yang2009topic} presents a four step method for doing the topic-level random walk to search scientific articles. 

Author: ```suggestion
\citet{yang2009topic} presents a method for doing the topic-level random walk to search scientific articles.
\todo[inline]{mention LM and reference section}.
```
this is never really referenced again is it? i cant remember anything about four steps.

They also describe various combinations of \gls{lda} and PageRank.
These combinations are evaluated against information retrieval baselines such as BM25\cite{bm251996}.

> While our method in some ways will be similar to the work of \citeauthor{Tang2008} and \citeauthor{yang2009topic}, our method will have a significant change of working with a new dataset consisting of documents without explicit connections between documents, namely news articles.

Author: ```suggestion
This paper is largely based on the work of \citeauthor{yang2009topic}\cite{yang2009topic} and \citeauthor{Tang2008}\cite{Tang2008}.
However, the novelty in this paper comes from a different focus as described by our goals in \todo{ref introduction / goals}.
For example, our method will have the significant change of working with a new dataset consisting of documents without explicit connections between documents, which introduces a challenge in using PageRank effectively.
```

Reviewer: we also differentiate our work from Tang et al. by automatically evaluating our IR performance by generating queries from our dataset

> \todo[inline]{we might need to focus on only applying LM*LDA*PageRank to the new dataset}

Author: ```suggestion
```

> This will be detailed further in the following sections.

Author: ```suggestion
```
